transport/server.cc

```diff
@@ -201,7 +201,7 @@ void cql_sg_stats::register_metrics()
 {
     namespace sm = seastar::metrics;
     std::vector<sm::metric_definition> transport_metrics;
-    auto& cur_sg_name = current_scheduling_group().name();
+    auto cur_sg_name = current_scheduling_group().name();
```

Comment:
I usually use `auto&&` which either grabs a reference, or extends the lifetime of a temporary, whichever works.

Reply:
thanks, I found both usages so used probably simpler form. Will use the universal reference in future so we don't need to differentiate the reference from the plain value.